tutorials/btp-hyperscaler-extension/btp-hyperscaler-extension.md

```diff
@@ -2,8 +2,8 @@
 parser: v2
 auto_validation: true
 time: 45
-tags: [ tutorial>intermediate, products>products>sap-btp--cloud-foundry-environment\, kyma-runtime]
-primary_tag: products>sap-btp--cloud-foundry-environment
+tags: [ tutorial>intermediate, software-product>sap-business-technology-platform\, kyma-runtime]
```

Comment:
Please remove backslash and correct tag for kyma runtime (software-product>sap-btp--kyma-runtime)

Reply:
@MichaelCzcz done